packages/data-transport-layer/src/services/main/service.ts

```diff
@@ -27,6 +27,7 @@ export interface L1DataTransportServiceOptions {
   l2RpcProvider: string
   l2RpcProviderUser?: string
   l2RpcProviderPassword?: string
+  l1SyncShutoffBlock?: number
```

Comment:
maybe we should make it configable?: https://github.com/ethereum-optimism/optimism/blob/df11f76b08a70c5a084f2d726d54ba4aa27bb7cc/packages/data-transport-layer/src/services/run.ts#L32